acceptance-tests/src/test/java/uk/gov/di/test/pages/EnterYourEmailAddressToBasePagePage.java

```diff
@@ -13,4 +13,9 @@ public void enterEmailAddressAndContinue(String emailAddress) {
         enterEmailAddress(emailAddress);
```

Comment:
Name of this page has gone a bit wrong...should be ...EnterYourEmailAddressToSignInPage, not EnterYourEmailAddressToBASEPAGEPage!

Also noticed that the CreateOrSignInPage currently has the same problem and is called CreateOrBASEPAGEPage!

A find and replace special!? Think that might have been like that for a couple of weeks tbf